packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -49,26 +61,42 @@ abstract class InAppPurchaseConnection {
   /// Returns true if the payment platform is ready and available.
   Future<bool> isAvailable();
 
-  /// Query product details list that match the given set of identifiers.
+  /// Query product details for the given set of IDs.
+  ///
+  /// [identifiers] needs to exactly match whatever the product IDs have been
```

Comment:
nit: never starts with a lower case word.
```suggestion
  /// The [identifiers] needs to exactly match whatever the product IDs have been
```